guides/representative-selection.md

```diff
@@ -1,16 +1,16 @@
 # Representative selection process
 
-This document is a guide for the selection of Representatives for the Leadership Council. This gives recommendations for how a top-level team should select their Representative. It is not a requirement that teams follow this guide, and may choose their own process.
+This document is a guide for the selection of Representatives for the Leadership Council. This gives recommendations for how a top-level team should select their Representative. It is not a requirement that teams follow this guide; teams may choose their own process.
```

Comment:
Given the clarification added, perhaps this should now be:

> ; top-level teams may choose their own process.

I wouldn't want to suggest that any team many choose their own process, in conflict with their parent top-level team.